aws-lambda-java-serialization/pom.xml

```diff
@@ -266,6 +266,28 @@
                     </execution>
                 </executions>
             </plugin>
-        </plugins>
-    </build>
+      <plugin>
+        <groupId>org.moditect</groupId>
+        <artifactId>moditect-maven-plugin</artifactId>
+        <version>1.0.0.Final</version>
```

Comment:
Latest version seems to be 1.2.2, could you please update it?
https://mvnrepository.com/artifact/org.moditect/moditect-maven-plugin/1.2.2.Final